crates/network/src/events/autonat.rs

```diff
@@ -12,51 +12,49 @@ impl EventHandler<Event> for EventLoop {
         match event {
             Event::OutboundProbe(outbound_probe_event) => match outbound_probe_event {
                 OutboundProbeEvent::Request { peer, .. } => {
-                    debug!("Sent probe request to: {:?}", peer)
+                    debug!(%peer, "Sent probe request");
                 }
                 OutboundProbeEvent::Response { peer, address, .. } => {
-                    debug!("Peer: {} determined external address {}", peer, address);
+                    debug!(%peer, %address, "Peer determined our external address");
 
                     let rendezvous_peers: Vec<PeerId> =
                         self.discovery.state.get_rendezvous_peer_ids().collect();
                     let relay_peers: Vec<PeerId> =
                         self.discovery.state.get_relay_peer_ids().collect();
 
-                    if self.is_autonat_status_public()
-                        && self.swarm.behaviour().autonat.confidence()
-                            >= self.discovery.autonat_config.confidence_threshold
+                    if self.swarm.behaviour().autonat.confidence()
+                        >= self.discovery.autonat_config.confidence_threshold
                     {
-                        for peer_id in &rendezvous_peers {
-                            if let Err(err) = self.rendezvous_discover(peer_id) {
-                                error!(%err, "Failed to perform rendezvous discovery");
-                            }
-                            if let Err(err) = self.rendezvous_register(peer_id) {
-                                error!(%err, "Failed to register with rendezvous");
+                        if self.is_autonat_status_public() {
```

Comment:
instead of indenting, short-circuit

```suggestion
                    if self.swarm.behaviour().autonat.confidence()
                        < self.discovery.autonat_config.confidence_threshold
                    {
                        return;
                    }
                    
                    if self.is_autonat_status_public() {
```